Orange/widgets/data/owimpute.py

```diff
@@ -348,6 +376,20 @@ def imputed_vars_for_column(self, column):
         else:
             assert False
 
+    def _handle_commit(func):
```

Comment:
Still somewhat undecided on the decorator name and whether comments are perhaps required..